next.config.js

```diff
@@ -11,8 +11,12 @@ const globalSassImports = `\
     @import "styles/mixins";
 `;
 
+function identity(value) {
+    return value;
+}
+
 const moduleExports = withBundleAnalyzer(
-    withPrezlyConfig({
+    identity({
```

Comment:
I've temporarily disabled this config to make sure it does not interfere with the new logic.